v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToSpanner.java

```diff
@@ -810,4 +865,30 @@ private static DeadLetterQueueManager buildDlqManager(Options options) {
       return DeadLetterQueueManager.create(dlqDirectory, retryDlqUri, 0);
     }
   }
+
+  private static ISchemaOverridesParser configureSchemaOverrides(Options options) {
+    // incorrect configuration
+    if (!options.getSchemaOverridesFilePath().isEmpty()
```

Comment:
can you add UT coverage for this ?

Reply:
This is part of `DatastreamToSpanner` class which initializes the beam pipeline. 
We don't seem to have UTs for this type of code - 

https://app.codecov.io/gh/GoogleCloudPlatform/DataflowTemplates/blob/main/v2%2Fdatastream-to-spanner%2Fsrc%2Fmain%2Fjava%2Fcom%2Fgoogle%2Fcloud%2Fteleport%2Fv2%2Ftemplates%2FDataStreamToSpanner.java

https://app.codecov.io/gh/GoogleCloudPlatform/DataflowTemplates/blob/main/v2%2Fsourcedb-to-spanner%2Fsrc%2Fmain%2Fjava%2Fcom%2Fgoogle%2Fcloud%2Fteleport%2Fv2%2Ftemplates%2FSourceDbToSpanner.java